src/webresource-handler/index.ts

```diff
@@ -67,6 +67,7 @@ export const getWebresourceHandler = (): WebResourceHandler | undefined => {
 const isFileInValidPath = async (
 	fieldname: string,
 	req: Express.Request,
+	odataRequest: uriParser.ParsedODataRequest,
 ): Promise<boolean> => {
 	if (req.method !== 'POST' && req.method !== 'PATCH') {
```

Comment:
Do we allow multipart requests on requests that aren't POST/PATCH?

Reply:
@Page- We don't do anything, basically. Whatever is the content of multpart/body in those requests is just dropped. IMHO it is the expected behaviour as it is the same thing of a GET request with a body (it works, but the body is just ignored as it should not be there).